Lib/unittest/mock.py

```diff
@@ -57,12 +56,12 @@ def _is_async_obj(obj):
         return False
     if hasattr(obj, '__func__'):
         obj = getattr(obj, '__func__')
-    return iscoroutinefunction(obj) or inspect.isawaitable(obj)
+    return inspect.iscoroutinefunction(obj) or inspect.isawaitable(obj)
```

Comment:
This increases the diff a lot, you should just import `iscoroutinefunction` directly and avoid all these changes/

Reply:
👌